### PR TITLE
Fix infrastractural problems

### PR DESCRIPTION
**Pull request by andreasonny83**
- Commit VSCode settings to enable Prettier on `ts` and `tsx` files
- Fix missing declarations in Storybook files that caused the develop script to fail
- Add `@gitgraph/core` as a direct dependency of `@gitgraph/stories`
- Fix `tsconfig` typo
- Add sleep to storybook script to fix async issues caused by `@gitgraph/core` not available

### CODE REVIEW

File: packages/stories/src/gitgraph-js/6-custom-renders.stories.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const withoutBranchLabels = templateExtend(TemplateName.Metro, {
   branch: { label: { display: false } },
 });
 
-const renderSax: CommitOptions["renderDot"] = (commit) =>
+const renderSax: CommitOptions["renderDot"] = (commit: any) =>
```
Comment (nicoespeon) [inline packages/stories/src/gitgraph-js/6-custom-renders.stories.tsx:28]: How come the type isn't inferred by TS here? Did you get failures?

File: packages/stories/package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "build": "build-storybook"
   },
   "dependencies": {
+    "@gitgraph/core": "1.4.5",
```
Comment (nicoespeon) [inline packages/stories/package.json:20]: Oh you're right, we depend directly on the Core package. See https://github.com/nicoespeon/gitgraph.js/blob/884f8496fcf754217ca7e067820ed71b4b88693d/packages/stories/src/gitgraph-react/1-basic-usage.stories.tsx#L4

Good catch 👍

File: packages/stories/.storybook/tsconfig.json [hunk 3]
```diff
@@ -1,22 +0,0 @@
-{
```
Comment (nicoespeon) [inline packages/stories/.storybook/tsconfig.json:1]: Was it useless?

File: package.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
     "lerna": "lerna",
     "build": "lerna run build --scope=@gitgraph/{core,react,js,node} --stream",
     "watch": "lerna run watch --stream",
-    "storybook": "lerna run storybook --stream",
+    "storybook": "sleep 3 && lerna run storybook --stream",
```
Comment (nicoespeon) [inline package.json:27]: This may look mysterious in the future, but hopefully this PR description will bring the relevant context behind this 😉

**Review by nicoespeon — APPROVED** [review]

File: packages/stories/.storybook/tsconfig.json (hunk 3 above)
Comment (andreasonny83) [reply packages/stories/.storybook/tsconfig.json:1]: yes, there was no typescript files in that folder

File: package.json (hunk 4 above)
Comment (andreasonny83) [reply package.json:27]: yep, that's actually a common problem when using concurrently and still need one task to be completed before the second one

**Pull request merged**